lib/nexmo_developer/app/controllers/application_controller.rb

```diff
@@ -32,6 +34,25 @@ def authenticate_admin!
     redirect_to root_path unless current_user.admin?
   end
 
+  def redirect_vonage_domain
+    return unless Rails.env.production? || Rails.env.test?
+
+    check_redirect_for(request)
+  end
+
+  def check_redirect_for(request)
```

Comment:
Could I suggest renaming this method to `process_redirect_for` rather than `check...`? (it's not really a check)